internal/levee_test.go

```diff
@@ -29,10 +30,18 @@ var patterns = []string{
 	"example.com/tests/sinks",
 }
 
+var debugging bool
+
 func TestLevee(t *testing.T) {
 	dir := analysistest.TestData()
 	if err := Analyzer.Flags.Set("config", dir+"/test-config.json"); err != nil {
 		t.Error(err)
 	}
+	if debugging {
+		analysistest.Run(t, dir, debug.Analyzer, patterns...)
+		// return without running the actual Analyzer since the above run
+		// does not produce any diagnostics, so the output will be very cluttered
+		return
```

Comment:
When `debugging` is `true`, running test emits failure results since no findings are emitted.  Do not early return here.  Or make it its own test, as recommended above.